src/marionnette_extensions/animatable_region.coffee

```diff
@@ -72,7 +71,7 @@ class AnimatableRegion extends Marionette.Region
 
       # use the transition that got us here, unless transitions were overridden
       if ! @navigationOptions?.transition?
-        @transition = previousNavigationItem?.transition || @transition
+        @transition = @currentPage?.transition || previousNavigationItem?.transition
```

Comment:
Unfortunately this breaks cases where the default page transition is overridden, because it now prefers the transition of the current page over the transition that brought in the current page.